templates/_components/card-icon.twig

```diff
@@ -0,0 +1,45 @@
+{% from '_macros/icon.twig' import icon %}
+
+{# @var string|null class #}
+{% set class = class ?? null %}
+
+{# @var string|null href #}
+{% set href = href ?? null %}
+
+{# @var string align #}
+{% set align = align ?? 'left' %}
+
+{# @var string|null icon #}
+{% set icon = icon ?? null %}
+
+{# @var string|null title #}
+{% set title = title ?? null %}
+
+{# @var string|null description #}
+{% set description = description ?? null %}
+
+
+<article class="{{ cx('flex flex-col gap-24', {
+  'items-start': align == 'left',
+  'items-center text-center': align == 'center',
+}, class) }}">
+
+  {% if icon %}
+    <div class="flex size-48 items-center justify-center rounded-full bg-sky-200 text-black">
+      {{ icon({
+        name: icon,
+      }) }}
+    </div>
+  {% endif %}
+
+  {% if title or description %}
+    <div class="flex flex-col gap-8">
+      {% if title %}
+        <h2 class="text-xl font-bold">{{ title }}</h2>
```

Comment:
This H2 should could probably be a variable. I'm also wondering if H3 is a better default if a typical page structure looks like:

```
H1 Page Title
   |- H2 My Card List
      |- H3 Card Title
      |- H3 Card Title
      |- H3 Card Title

Reply:
Good call! I'll update in my next PR